tests/Nox.Generator.Tests/Common/CompileTests.cs

```diff
@@ -94,6 +94,12 @@ private static IEnumerable<MetadataReference> GetReferences()
         {
```

Comment:
all the three foreach and the dllFiles calculation could be encapsulated in a single method